fhd_core/polarization/stokes_cnv.pro

```diff
@@ -172,6 +173,7 @@ ENDIF ELSE BEGIN ;else case is array of images
     ;stokes I can have proper inverse-variance weighting. (not used!)
     ; All other polarizations need to be converted to 'true sky' frame before they can be added
     IF Keyword_Set(inverse) THEN BEGIN ;Stokes -> instrumental
+        sign=[1,-1,-complex(0,1),complex(0,1)]
```

Comment:
For consistency, let's make this sign1 = [1, 1, 1, 1] and sign2=[1,-1,-complex(0,1),complex(0,1)], and use both on line 178 below.